docs/data/base/components/number-input/NumberInputIntroduction.js

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import NumberInput, { numberInputClasses } from '@mui/base/Unstable_NumberInput';
+import { styled } from '@mui/system';
+
+const CustomNumberInput = React.forwardRef(function CustomNumberInput(props, ref) {
```

Comment:
It's most important to have the Tailwind + Plain CSS demos on the introduction demo :) It's the first thing people see when they come to the page :)